tools/clang/test/CodeGenSPIRV/op.struct.access.bitfield.sized.hlsl

```diff
@@ -0,0 +1,148 @@
+// RUN: %dxc -T cs_6_2 -E main -spirv -fcgl -enable-16bit-types %s | FileCheck %s
+
+struct S1 {
+  uint64_t f1 : 32;
+  uint64_t f2 : 1;
+};
+// CHECK-DAG: %S1 = OpTypeStruct %ulong
+
+struct S2 {
+  uint16_t f1 : 4;
+  uint16_t f2 : 5;
+};
+// CHECK-DAG: %S2 = OpTypeStruct %ushort
+
+struct S3 {
+  uint64_t f1 : 45;
+  uint64_t f2 : 10;
+  uint16_t f3 : 7;
+  uint32_t f4 : 5;
+};
+// CHECK-DAG: %S3 = OpTypeStruct %ulong %ushort %uint
+
+struct S4 {
+  int64_t f1 : 32;
+  int64_t f2 : 1;
+};
+// CHECK-DAG: %S4 = OpTypeStruct %long
+
+[numthreads(1, 1, 1)]
+void main() {
+  S1 s1;
+  s1.f1 = 3;
+// CHECK:   [[ptr:%[0-9]+]] = OpAccessChain %_ptr_Function_ulong %s1 %int_0
+// CHECK:   [[tmp:%[0-9]+]] = OpLoad %ulong [[ptr]]
+// CHECK:  [[mask:%[0-9]+]] = OpShiftLeftLogical %ulong %ulong_4294967295 %uint_0
```

Comment:
Can you add comments with the binary/hex notation for all the values and masks? Without that it's hard to verify that any of these values are correct.